src/Moryx/Configuration/PossibleValuesSerialization.cs

```diff
@@ -93,7 +93,7 @@ public override string[] PossibleElementValues(Type memberType, ICustomAttribute
 
             // Use attribute
             var values = valuesAttribute.GetValues(Container, ServiceProvider);
```

Comment:
Do we need to enhance the API of `GetValues` on the attribute or the Attribute itself as well?